### PR TITLE
Openapi-Merger Rename

### PR DESCRIPTION
**Pull request by nitro-marky**
Hopefully this has renamed all of the old wasp names with the new one. This has only changed names,  any code changes will need to be done seperately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Install `minikube` and `helm` using Homebrew, then start `minikube` and update h
 brew install minikube helm
 minikube start --vm=true --driver=hyperkit
 minikube addons enable ingress
-helm dependency update helm/wasp-open-api
+helm dependency update helm/openapi-merger
```
Comment (mattdean-digicatapult) [inline README.md:49]: I believe this section on `Helm/Kubernetes` is now redundent @dblane-digicatapult and should be removed?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@digicatapult/wasp-open-api",
-  "version": "0.1.50",
+  "name": "@digicatapult/openapi-merger",
+  "version": "0.0.1",
```
Comment (mattdean-digicatapult) [inline package.json:3]: Can we make this version `1.0.0` please

**Review by mattdean-digicatapult — COMMENTED** [review]
There's a few more `WASP` references to remove I think. Most of these can be replaced with `openapi-merger` with the following exceptions:

1. In `package.json` there's a keywords section. Remove `wasp` and add in `OpenAPI`
2. In the release workflow there's a couple of secrets. You should ask @dblane-digicatapult about the secrets to use

**Comment by jonmattgray** [comment]
the repo is still using the old check-version script so that can be replaced with the new action

**Comment by nitro-marky** [comment]
> the repo is still using the old check-version script so that can be replaced with the new action

Done

File: README.md (hunk 1 above)
Comment (nitro-marky) [reply README.md:49]: > I believe this section on `Helm/Kubernetes` is now redundent @dblane-digicatapult and should be removed?

Left it for now

**Comment by nitro-marky** [comment]
> There's a few more `WASP` references to remove I think. Most of these can be replaced with `openapi-merger` with the following exceptions:
> 
> 1. In `package.json` there's a keywords section. Remove `wasp` and add in `OpenAPI`
> 2. In the release workflow there's a couple of secrets. You should ask @dblane-digicatapult about the secrets to use

Left release for now

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -94,19 +94,16 @@ jobs:
     name: "Check version"
     runs-on: ubuntu-latest
 
+  check-version:
+    name: "Check version"
+    runs-on: ubuntu-latest
```
Comment (jonmattgray) [inline .github/workflows/tests.yml:99]: got same code twice
```
 check-version:
    name: "Check version"
    runs-on: ubuntu-latest
 ```

File: README.md (hunk 1 above)
Comment (dblane-digicatapult) [reply README.md:49]: Yeah can nuke this

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
There are still quite a few mentions of WASP, almost entirely in the markdown files and in the PR/Bug templates in the .github folder.  Please remove these.

**Comment by dblane-digicatapult** [comment]
Please read those changes you made and ensure they make sense. Half of them do not. If there is casing then there is probably a reason for it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ about: Create a report to help us improve
 
 <!--
 
-Have you read WASP's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/wasp-open-api/.github/blob/main/CODE_OF_CONDUCT.md
+Have you read openapi's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/openapi-merger/.github/blob/main/CODE_OF_CONDUCT.md
```
Comment (dblane-digicatapult) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]:
```suggestion
Have you read Digital Catapult's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/openapi-merger/.github/blob/main/CODE_OF_CONDUCT.md
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ about: Suggest an idea for this project
 
 <!--
 
-Have you read WASP's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/wasp-open-api/.github/blob/main/CODE_OF_CONDUCT.md
+Have you read openapi's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/openapi-merger/.github/blob/main/CODE_OF_CONDUCT.md
```
Comment (dblane-digicatapult) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]:
```suggestion
Have you read Digital Catapult's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/openapi-merger/.github/blob/main/CODE_OF_CONDUCT.md
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ Also note that the Digital Catapult team has finite resources so it's unlikely t
 
 ## Describe alternatives you've considered
 
-<!-- A clear and concise description of the alternative solutions you've considered. Be sure to explain why WASP's existing customisability isn't suitable for this feature. -->
+<!-- A clear and concise description of the alternative solutions you've considered. Be sure to explain why openapi's existing customisability isn't suitable for this feature. -->
```
Comment (dblane-digicatapult) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]:
```suggestion
<!-- A clear and concise description of the alternative solutions you've considered. Be sure to explain why openapi-merger's existing customisability isn't suitable for this feature. -->
```

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -175,8 +168,8 @@ jobs:
       - name: Login to Dockerhub Registry
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.WASP_DOCKERHUB_USERNAME }}
-          password: ${{ secrets.WASP_DOCKERHUB_TOKEN }}
+          username: ${{ secrets.openapi_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.openapi_DOCKERHUB_TOKEN }}
```
Comment (dblane-digicatapult) [inline .github/workflows/release.yml:172]: Leave as WASP, no point creating a new token just for this.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,8 +1,8 @@
-# Contributing to wasp-open-api
+# Contributing to openapi-merger
 
 Firstly, we would like to thank you for taking the time to contribute!
 
-The following is a set of guidelines for contributing to WASP and its packages, which are hosted on the [digicatapult](https://github.com/digicatapult) organisation on GitHub. These are mostly guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to openapi and its packages, which are hosted on the [digicatapult](https://github.com/digicatapult) organisation on GitHub. These are mostly guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
```
Comment (dblane-digicatapult) [inline CONTRIBUTING.md:5]:
```suggestion
The following is a set of guidelines for contributing to openapi-merger and its packages, which are hosted on the [digicatapult](https://github.com/digicatapult) organisation on GitHub. These are mostly guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
```

File: README.md [hunk 9]
```diff
@@ -1,17 +1,18 @@
-# wasp-open-api
+# openapi-merger
 
-Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of wasp OpenAPI specs defined under `paths` in [`values.yaml`](helm/wasp-open-api/values.yaml).
+Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of openapi OpenAPI specs defined under `paths` in [`values.yaml`](helm/openapi-merger/values.yaml).
```
Comment (dblane-digicatapult) [inline README.md:3]: Link to the wasp-openapi-merger in the helm repo rather than to a non-existent path

File: README.md [hunk 10]
```diff
@@ -1,17 +1,18 @@
...
 
 Includes a node service with three endpoints:
+
 - GET `/api-docs` to access the merged API spec
 - `/swagger` to view the merged API spec in Swagger
 - POST `/set-api-docs` to set the merged API spec
 
-The merge is automated via a Helm [`CronJob`](helm/wasp-open-api/templates/cronjob.yaml). that runs [`openapi-merge-cli`](https://www.npmjs.com/package/openapi-merge-cli) then posts the merged doc to `set-api-docs` every minute.
+The merge is automated via a Helm [`CronJob`](helm/openapi-merger/templates/cronjob.yaml). that runs [`openapi-merge-cli`](https://www.npmjs.com/package/openapi-merge-cli) then posts the merged doc to `set-api-docs` every minute.
```
Comment (dblane-digicatapult) [inline README.md:11]: Link to the correct path in the helm repo.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Comment by mattdean-digicatapult** [comment]
@dblane-digicatapult I think I agree with all these suggestions and they are sensible. They are however in conflict with our open-sourcing template materials which probably need reviewing. @dblane-digicatapult can you add an improvement ticket to SEI to cover making these guidelines more sensible. @nitro-marky I think you're free to to implement the suggestions @dblane-digicatapult has made

**Comment by dblane-digicatapult** [comment]
> @dblane-digicatapult I think I agree with all these suggestions and they are sensible. They are however in conflict with our open-sourcing template materials which probably need reviewing. @dblane-digicatapult can you add an improvement ticket to SEI to cover making these guidelines more sensible. @nitro-marky I think you're free to to implement the suggestions @dblane-digicatapult has made

Ticket created SEI-43

File: README.md [hunk 11]
```diff
@@ -1,8 +1,9 @@
-# wasp-open-api
+# openapi-merger
 
-Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of wasp OpenAPI specs defined under `paths` in [`values.yaml`](helm/wasp-open-api/values.yaml).
+Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of openapi OpenAPI specs defined under `paths` in [`values.yaml`](helm/wasp-open-api/values.yaml).
```
Comment (dblane-digicatapult) [inline README.md:3]:
```suggestion
Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of OpenAPI specs defined under `paths` in [`values.yaml`](https://github.com/digicatapult/helm-charts/blob/main/charts/wasp-open-api/values.yaml).
```

File: package.json [hunk 12]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@digicatapult/wasp-open-api",
-  "version": "0.1.50",
-  "description": "Amalgamated API specs for WASP",
+  "name": "@digicatapult/openapi-merger",
+  "version": "1.0.1",
```
Comment (dblane-digicatapult) [inline package.json:3]:
```suggestion
  "version": "1.0.0",
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**